lib/resources/example_json/r4_examples_insurance_plan.rb

```diff
@@ -0,0 +1,141 @@
+                  }
+                ],
+                'text': 'Medical Service Plan'
+              }
+            ],
+            'name': 'Miscellaneous Commercial Health Plan',
+            'period': {
+              'start': '2003-08-26T00:00:00.000Z'
+            },
+            'ownedBy': {
+              'reference': 'Organization/589803'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir.applications.stagingcerner.com/r4/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/InsurancePlan/589724',
+          'resource': {
+            'resourceType': 'InsurancePlan',
+            'id': '589724',
+            'text': {
+              'status': 'generated',
+              'div': '<div xmlns=\'http://www.w3.org/1999/xhtml\'><p><b>Insurance Plan</b></p><p><b>Name</b>:' \
+                     'Miscellaneous Worker&#39;s Comp Plan</p><p><b>Type</b>: Medical Service Plan</p><p><b>Owned By' \
+                     '</b>: Organization/589743</p><p><b>Period Start Date</b>: Aug 26, 2003 12:00 A.M. UTC</p></div>'
+            },
+            'status': 'active',
+            'type': [
+              {
+                'coding': [
+                  {
+                    'system': 'https://fhir.test.cerner.com/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/codeSet/27137',
+                    'code': '6681216',
+                    'display': 'Medical Service Plan',
+                    'userSelected': true
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/insurance-plan-type',
+                    'code': 'medical',
+                    'display': 'Medical',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Medical Service Plan'
+              }
+            ],
+            'name': "Miscellaneous Worker's Comp Plan",
+            'period': {
+              'start': '2003-08-26T00:00:00.000Z'
+            },
+            'ownedBy': {
+              'reference': 'Organization/589743'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir.applications.stagingcerner.com/r4/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/InsurancePlan/794136',
+          'resource': {
+            'resourceType': 'InsurancePlan',
+            'id': '794136',
+            'text': {
+              'status': 'generated',
+              'div': '<div xmlns=\'http://www.w3.org/1999/xhtml\'><p><b>Insurance Plan</b></p><p><b>Name</b>:' \
+                     'RCR_HealthPlan_auto2</p><p><b>Type</b>: Medical Service Plan</p><p><b>Owned By</b>:' \
+                     'Organization/1930312</p><p><b>Period Start Date</b>: Jan 17, 2017  3:18 A.M. UTC</p></div>'
+            },
+            'status': 'active',
+            'type': [
+              {
+                'coding': [
+                  {
+                    'system': 'https://fhir.test.cerner.com/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/codeSet/27137',
```

Comment:
should this be a fhir.test.cerner.com domain or a fhir.cerner.com?

Reply:
In the actual staging validation it is fhir.test.cerner.com

Reply:
changed https://github.com/cerner/fhir.cerner.com/pull/809/files#diff-c55d86b9fc3035c325e102407d939636235d36c5397e8858e03b935a6613a874R18